phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionlessQueryServicesImpl.java

```diff
@@ -600,7 +604,12 @@ public String getUserName() {
 
     @Override
     public GuidePostsInfo getTableStats(GuidePostsKey key) {
-        GuidePostsInfo info = guidePostsCache.getCache().getIfPresent(key);
+        GuidePostsInfo info = null;
```

Comment:
Do we need to handle this exception in a similar way in `ConnectionQueryServicesImpl.java`?
We may not be able to move to parent class because I believe that Java 7 doesn't support default methods in interfaces, right? Do you think there is a better way?